player/src/main/java/net/pubnative/player/VASTPlayer.java

```diff
@@ -324,7 +325,11 @@ private void setEmptyState() {
         mQuartile = 0;
         mTrackingEventMap = null;
         mProgressTracker = null;
-        mBannerBitmap = null;
+        if (mIsBannerSet) {
```

Comment:
but @anshumansworld then the solution would be to nullify the bannerBitmap when you assign a new video no?